crypto/evp/pmeth_lib.c

```diff
@@ -833,6 +862,22 @@ static int legacy_ctrl_str_to_param(EVP_PKEY_CTX *ctx, const char *name,
         pad = atoi(value);
         return EVP_PKEY_CTX_set_dh_pad(ctx, pad);
     }
+# endif
+# ifndef OPENSSL_NO_EC
+    if (strcmp(name, "ecdh_cofactor_mode") == 0) {
+        int mode;
+
+        mode = atoi(value);
+        return EVP_PKEY_CTX_set_ecdh_cofactor_mode(ctx, mode);
+    }
+    if (strcmp(name, "ecdh_kdf_md") == 0) {
+        const EVP_MD *md;
+        if ((md = EVP_get_digestbyname(value)) == NULL) {
+            /* TODO(rome) raise error: EC_R_INVALID_DIGEST */
+            return 0;
+        }
+        return EVP_PKEY_CTX_set_ecdh_kdf_md(ctx, md);
+    }
```

Comment:
From `ec_pmeth.c` only these 2 had associated `ctrl_str`. Do we need to allocate strings for the other parameters for `ecdh_kdf`?

Reply:
This is about legacy support. If the legacy didn't have it we don't need to add it now.